lib/beaker/hypervisor/vcloud.rb

```diff
@@ -147,15 +154,31 @@ def provision
 
         try = (Time.now - start) / 5
         duration = run_and_report_duration do
-          @hosts.each_with_index do |h, i|
-            wait_for_dns_resolution(h, try, attempts)
+          @hosts.each do |host|
+            #wait_for_dns_resolution(h, try, attempts)
+            until @vsphere_helper.find_vms(host['vmhostname'])[host['vmhostname']].summary.guest.ipAddress != nil
```

Comment:
Is it possible that this `until` will never exit? Should we be taking a maximum number of attempts into account?